lib/fake_async.dart

```diff
@@ -98,9 +98,9 @@ class FakeAsync {
   ///
   /// Note: it's usually more convenient to use [fakeAsync] rather than creating
   /// a [FakeAsync] object and calling [run] manually.
-  FakeAsync({DateTime initialTime}) {
+  FakeAsync({DateTime? initialTime}) {
     initialTime ??= clock.now();
-    _clock = Clock(() => initialTime.add(elapsed));
+    _clock = Clock(() => initialTime!.add(elapsed));
```

Comment:
@leafpetersen - is there some opportunity for flow analysis here?

Is there some way to use `late` that would avoid the overhead in this callback?

We could potentially do it manually.

```dart
final nonNullInitialTime = initialTime ?? clock.now();
_clock = Clock(() => nonNullInitialTime.add(elapsed));
```

Reply:
I will go ahead and do the workaround for now. I did open an issue about `??=` on the sdk a while ago, https://github.com/dart-lang/sdk/issues/41172